lib/services/internalEngine.js

```diff
@@ -43,8 +42,8 @@ function InternalEngine (kuzzle) {
   this.init = function () {
     if (!this.client) {
       this.client = new Elasticsearch.Client({
-        host: kuzzle.config[this.engineType].hosts,
-        apiVersion: kuzzle.config[this.engineType].apiVersion
+        host: kuzzle.config.services.db.host + ':' + kuzzle.config.services.db.port,
```

Comment:
nitpicking: If I understand correctly the refactor of `lib/services/index.js`, we could also use `config` as a constructor's parameter (`function InternalEngine(kuzzle, options, config) {...}`), and use here:
```
this.client = new Elasticsearch.Client({
  host: config.host + ':' + config.port,
  apiVersion: config.apiVersion
});
```

(to be consistent with other services implementation)